terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/message/MessageTest.java

```diff
@@ -182,6 +182,74 @@ public void test01_07_defaultSpecified() {
     public void test01_08_defaultSpecified() {
         driver.findElement(By.id("defaultSpecified_01_08")).click();
 
+        // div ul li Tag confirm
+        // Message Confirm
+        assertThat(driver.findElement(By.xpath("/html/body/div/div/ul/li"))
+                .getText(), is("Hello World!!"));
+
+        // <div> Tag class is "alert alert-danger"
```

Comment:
- [ ] Did you mistakenly comment "alert-danger"?
(alert-primary / alert-secondary / alert-light / alert-dark)

Reply:
Sorry, I didn't notice it. https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/897/commits/5e16a6a7a9df9ee63138f4207eaf481ac90ebbce